### ISSUE TITLE
Selecting text resizes the window

### ISSUE THREAD

**Issue by sebastinas** [pwmt/girara#28]
*On GitLab by @peachey2k2 on Dec 23, 2023, 13:27*

---

Whenever I select a piece of text, it forcefully resizes the window vertically to fit that text in the bottom line where it says `Copied selected text to selection clipboard: {text}`. When it happens, the window can't be shrunken to a smaller size until I hit the escape key, which hides that bottom line. This happens even when the window is maximized and in fullscreen.

zathura --version output:
```
>> zathura --version
Gtk-Message: 15:09:09.790: Failed to load module "colorreload-gtk-module"
Gtk-Message: 15:09:09.791: Failed to load module "window-decorations-gtk-module"
zathura 0.5.4
girara 0.4.1 (runtime: 0.4.1)
(plugin) pdf-poppler (0.3.2) (/usr/lib/zathura/libpdf-poppler.so)
```
DE: Xfce 4.18

Video recording demonstrating the issue:

![untitled](https://git.pwmt.org/pwmt/girara/uploads/2149bc37e77c085bf6e2f4666d9d6882/untitled.mp4)

**Comment by sebastinas** [comment]
moved from zathura#394

**Comment by sebastinas** [comment]
Fixed in 87775a28e3a4195b19f0976b2c2d38b96c4b7fa0.
